deploy_win.py

```diff
@@ -266,7 +266,8 @@ def write_license(name, contents, url, collapse_double_newlines):
         ('ASI TICOFF', ('sub', 'epyqlib', 'epyqlib', 'ticoff.asi_license.txt'), 'https://gist.github.com/eliotb/1073231', False),
         ('pyelftools', (), 'https://raw.githubusercontent.com/eliben/pyelftools/master/LICENSE', False),
         ('attrs', ('installer', 'attrs.txt'), 'https://raw.githubusercontent.com/hynek/attrs/5516d5452c76416b3fea0c9d9f2b7505fae82c2b/LICENSE', False),
-        ('FontAwesome', (), 'https://scripts.sil.org/cms/scripts/render_download.php?format=file&media_id=OFL_plaintext&filename=OFL.txt', False),
+        # ('FontAwesome', (), 'https://scripts.sil.org/cms/scripts/render_download.php?format=file&media_id=OFL_plaintext&filename=OFL.txt', False),
+        ('FontAwesome', ('sub', 'epyqlib', 'epyqlib', 'fontawesome.ofl_license.txt'), None, False),
```

Comment:
(repeat as original got 'outdated')

This was for a temporary issue downloading from the URL, right?  I mean, maybe the files should all be stored locally instead of acquiring them, and maybe we should use https://pypi.org/project/spdx-tools/ or somesuch helpful tool.  Just making sure I understand the why.

Reply:
(Copying response here)

> This was for a temporary issue downloading from the URL, right?

That's my recollection, yes. 

> I mean, maybe the files should all be stored locally instead of acquiring them

That's what I've done, at least with this file which does not appear to be anything that will change ever. So checking it in directly instead of fetching it every time seems safe to me.